src/main/kotlin/org/jetbrains/bio/genome/AnnotationsConfig.kt

```diff
@@ -159,7 +165,8 @@ object AnnotationsConfig {
             try {
                 return GenomeAnnotationsConfig(
                     genomeAttrs["species"] as String,
-                    genomeAttrs["alias"] as String?,
+                    genomeAttrs["ucsc_alias"] as? String,
```

Comment:
Is `ucsc_alias` obligatory or not? If yes, we could expect non nullable string here

Reply:
I'm fairly sure there is a genome assembly out there that doesn't have a UCSC equivalent. So I've decided to make it optional.